objects-revisited/main.js

```diff
@@ -0,0 +1,123 @@
+const business = {
+  opens: '7am',
+  closes: '8pm',
+  totalEmployees: 4,
+  daysOpen: ['M', 'Tu', 'W', 'Th', 'F'],
+  employees: {
+    marshall: {
+      position: 'c.e.o',
+      daysOfWeekWorking: ['Tu', 'W', 'Th'],
+    },
+    nicholas: {
+      position: 'marketing director',
+      daysOfWeekWorking: ['M', 'W', 'F'],
+    },
+    justin: {
+      position: 'administrative assistant',
+      daysOfWeekWorking: ['M', 'Tu', 'W'],
+    },
+    frankie: {
+      position: 'project manager',
+      daysOfWeekWorking: ['W', 'Th', 'F'],
+    },
+  },
+};
+
+const { daysOpen, employees } = business;
+const employeePositions = [
+  'salesperson',
+  'software engineer',
+  ' c.f.o.',
+  'c.o.o.',
+];
+
+// add weekends to employee schedules
+function addWeekends(object) {
```

Comment:
The name `object` is so general that it's hard to understand what it's supposed to be. In fact, I had to search the code to figure it out! The name `employees` would be much more clear.